src/airbrake.cr

```diff
@@ -49,22 +49,29 @@ module Airbrake
   class Config
     property :project_id
     property :project_key
+    property :host
+    property :port
+    property :secure
+
+    def uri
+      self.host   ||= "airbrake.io"
+      self.port   ||= 443
+      scheme = (self.secure.nil? || self.secure) ? "https" : "http"
+
+      URI.new(scheme, host, port, "/api/v3/projects/#{project_id}/notices", "key=#{project_key}").to_s
```

Comment:
Yeah, that does make sense. It was just some pseudocode. Basically, I'd like to keep as little options as possible. If it's not possible to implement in Crystal, then fair enough. However, it would be really nice to have just 1 option instead of 3 :P